lib/Db/StackMapper.php

```diff
@@ -48,6 +48,25 @@ public function find($id): Stack {
 		return $this->findEntity($sql, [$id]);
 	}
 
+	/**
+	 * @param $cardId
+	 * @return Stack|null
+	 */
+	public function findStackFromCardId($cardId): ?Stack {
+		$sql = <<<SQL
```

Comment:
Fine for now, but in general we should also aim for moving this class to inherit from QBMapper instead of Mapper so we can easily use the query builder API here.